### PR TITLE
Restructure website

### PR DESCRIPTION
**Pull request by aknierim**
This PR restructures the navbar (see #277) and also adds a footer to the bottom of the page.

### CODE REVIEW

**Comment by aknierim** [comment]
Some notes (will update if more comes to my mind):
* Footer still needs some work, i.e.:
  - sticking it to the bottom of the page, regardless of page/screen size (see e.g. ``install/index/``)
  - centering of the elements is somewhat off for now
  - a button to the main pep website could be nice

File: _includes/footer.html [hunk 1]
```diff
@@ -0,0 +1,24 @@
+<div class="footer text-center mt-auto">
+  <ul>
+    <div class="mb-2">
+      <a href="https://www.facebook.com/groups/pep.dortmund/" role="button" class="btn btn-lg btn-outline-secondary">
+        <span class="fab fa-facebook"></span>
+      </a>
+      <a href="https://twitter.com/pepdortmund" role="button" class="btn btn-lg btn-outline-secondary">
+        <span class="fab fa-twitter"></span>
```
Comment (chrbeckm) [inline _includes/footer.html:8]: Do we still use that account?

Comment (chrbeckm) [reply _includes/footer.html:8]: Maybe better to link Linkedin: https://www.linkedin.com/company/pep-et-al

Comment (aknierim) [reply _includes/footer.html:8]: > Do we still use that account?

Latest post from 2022, so I guess not... Will add the LinkedIn URL instead 👍

**Review by chrbeckm — APPROVED** [review]

**Pull request merged**